### PR TITLE
Election name

### PR DESCRIPTION
**Pull request by cikzh**
Small PR that fills the Election name in the Input home page. Couldn't find an issue addressing this, but I think this would fit nicely into #96

### CODE REVIEW

**Comment by praseodym** [comment]
This is listed in the description of #108 :)

**Review by praseodym — DISMISSED** [review]

**Review by Lionqueen94 — APPROVED** [review]

File: frontend/app/module/input/page/InputHomePage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import { overrideOnce, render, screen } from "app/test/unit";
+
+import { ElectionProvider } from "@kiesraad/api";
+import { electionMock } from "@kiesraad/api-mocks";
+
+import { InputHomePage } from "./InputHomePage";
+
+describe("InputHomePage", () => {
+  overrideOnce("get", "/api/elections/1", 200, { election: electionMock });
```
Comment (jschuurk-kr) [inline frontend/app/module/input/page/InputHomePage.test.tsx:11]: We have an `electionMockData` and a `electionsMockData` in `frontend/lib/api-mocks/ElectionMockData.ts`. They both have an election with id `1`, but their `name`s are different (no differences in other fields). Would be better if the `name`s match.

Comment (Lionqueen94) [reply frontend/app/module/input/page/InputHomePage.test.tsx]: I'll update this in my upcoming PR. I've already been making changes to the mocks

Comment (cikzh) [reply frontend/app/module/input/page/InputHomePage.test.tsx]: I've already pushed a commit. Should I revert it to make the rebase easier?

Comment (Lionqueen94) [reply frontend/app/module/input/page/InputHomePage.test.tsx]: No it's okay, I'll figure it out :+1:

**Review by Lionqueen94 — APPROVED** [review]

File: frontend/app/module/input/page/InputHomePage.test.tsx (hunk 1 above)
Comment (cikzh) [reply frontend/app/module/input/page/InputHomePage.test.tsx]: @Lionqueen94 is what @jschuurk-kr suggests something the mock changes will cover?

Comment (Lionqueen94) [reply frontend/app/module/input/page/InputHomePage.test.tsx]: Yes that will be done, I'll be showing the election name in the header as per design in the input pages

Comment (cikzh) [reply frontend/app/module/input/page/InputHomePage.test.tsx]: Cool, I'll resolve this

Comment (praseodym) [reply frontend/app/module/input/page/InputHomePage.test.tsx]: > Or just share the same mock object?

Something like this: https://gist.github.com/praseodym/cf70d91d6ea7ecc05138963fbcea363e

File: frontend/lib/api-mocks/ElectionMockData.ts [hunk 2]
```diff
@@ -210,7 +210,7 @@ export const politicalGroupMockData: PoliticalGroup = {
   ],
 };
 
-export const electionMockData: Election = {
+export const electionDetailMockData: Election = {
   id: 1,
   name: "Municipal Election",
```
Comment (Lionqueen94) [inline frontend/lib/api-mocks/ElectionMockData.ts:215]:
```suggestion
  name: "Gemeenteraadsverkiezingen 2026",
```

File: frontend/app/module/input/page/InputHomePage.test.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+describe("InputHomePage", () => {
+  overrideOnce("get", "/api/elections/1", 200, { election: electionDetailMock });
+  test("Election name", async () => {
+    render(
+      <ElectionProvider electionId={1}>
+        <InputHomePage />
+      </ElectionProvider>,
+    );
+
+    expect(await screen.findByText("Municipal Election")).toBeVisible();
```
Comment (Lionqueen94) [inline frontend/app/module/input/page/InputHomePage.test.tsx:19]:
```suggestion
    expect(await screen.findByText(electionDetailMock.name)).toBeVisible();
```

File: frontend/app/module/input/page/InputHomePage.test.tsx (hunk 1 above)
Comment (Lionqueen94) [reply frontend/app/module/input/page/InputHomePage.test.tsx]: These 2 commits fixed the name inconsistency. My PR will make the file a lot cleaner including Mark's suggestion

**Review by Lionqueen94 — APPROVED** [review]

**Review by jschuurk-kr — APPROVED** [review]

**Pull request merged**